conda/bld.bat

```diff
@@ -0,0 +1,8 @@
+COPY options_conda_win.py options.py
+
+REM scons install
+python setup.py install
+
+if errorlevel 1 exit 1
+
+MOVE %LIBRARY_PREFIX%\lib\*pgl*.dll*  %LIBRARY_PREFIX%\bin
```

Comment:
The MOVE is cleaver. Good point.
Do you think we have to move all the dll in bin for every projects?